capstone/src/main/java/com/google/sps/servlets/AssociationServlet.java

```diff
@@ -35,17 +35,22 @@ public void init() throws ServletException {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(OUTPUT_TYPE);
 
-    FetchOptions fetchOptions = FetchOptions.Builder.withLimit(LIMIT);
+    String scope = request.getParameter("scope");
+    if (scope == null) {
+      scope = "SF";
```

Comment:
Is there a way that you and Nitin could both use a server to get the value of "scope" or "precinct"? (If it becomes unlikely that a query will work.)

Reply:
The main problem with this is that then you would have to store it by users in order to allow different users to be on different precincts, which seems overcomplicated. I think its better to store state such as precinct on the client side.